rke/templates/templates.go

```diff
@@ -78,7 +78,8 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			">=1.16.4-rancher1 <1.16.8-rancher0":       canalv117,
 			">=1.16.0-alpha <1.16.4-rancher1":          canalv116,
 			">=1.15.12-rancher2-2 <1.16.0-alpha":       canalv115PrivilegedCalico3134,
-			">=1.15.11-rancher1-1 <1.15.12-rancher2-2": canalv115Privileged,
+			">=1.15.11-rancher1-1 <1.15.12-rancher1-1": canalv115Privileged,
+			">=1.15.12-rancher1-1 <1.15.12-rancher2-2": canalv115,
```

Comment:
Why were we using `canalv115Privileged` and now we're not for 1.15.12?

Reply:
Added comment to clarify this logic 
```
// 1.15.12-rancher1-1 comes from 2.2.13, uses old canalv115 template
```